src/api/todos.js

```diff
@@ -0,0 +1,76 @@
+// Todos table functions 
+import axios from './axios';
+
+// RESTful API endpoint for todo objects
+const endpoint = '/todos';
+
+/**
+ * Get all rows in the todos table.
+ * @returns array of todos 
+ */
+async function getTodos() {
```

Comment:
I would suggest having the full API response, response headers + body to be passed to the service caller e.g. from a component. We will need the response headers to handle 4xx/5xx errors on the caller i.e. try-catch.

Reply:
I'll return status and header information also for all API requests.